app/controllers/auctions_controller.rb

```diff
@@ -1,13 +1,13 @@
 class AuctionsController < ApplicationController
   def index
     collection = AuctionQuery.new.public_index
-    @view_model = AuctionsIndexViewModel.new(current_user, collection)
+    @auctions = AuctionsIndexViewModel.new(auctions: collection, current_user: current_user)
```

Comment:
ok cool -- how about tabling for now but as I move to next refactor I will change to `user` ?